sdk/rust/Cargo.toml

```diff
@@ -3,8 +3,17 @@ name = "spin-sdk"
 version = { workspace = true }
 authors = { workspace = true }
 edition = { workspace = true }
+license = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+documentation = "https://fermyon.github.io/rust-docs/spin/main/spin_sdk"
```

Comment:
Is there a reason we prefer this over the docs.rs hosting?

Reply:
Oh good point, will update to https://docs.rs/spin-sdk

Reply:
Yea which redirects to https://docs.rs/spin-sdk/latest/spin_sdk